### PR TITLE
APP-4609 feat(crop content): add collapsed control

### PR DESCRIPTION
**Pull request by AliciaSymphony**
## Ticket
https://perzoinc.atlassian.net/browse/APP-4609

## Description
- Add a control of the collapse logic from external
- Refine the storybook doc
- Add a storybook example

## Demo
Nov-30-2021 15-48-41

### CODE REVIEW

**Review by symphony-annaLlorens — APPROVED** [review]

**Comment by symphony-adnane** [comment]
I'd call the documentation button "toggle" instead of reverse XD

File: src/components/crop-content/CropContent.tsx [hunk 1]
```diff
@@ -10,6 +10,8 @@ type CropContentProps = {
   className?: string;
   /** If true, the crop will be collapsed initially. */
   initCollapsed?: boolean;
+  /** Control the collapsed state, otherwise it is uncontrolled */
+  collapsed?: boolean;
```
Comment (symphony-adnane) [inline src/components/crop-content/CropContent.tsx:14]: is it possible to deprecate maybe initCollapsed?

**Review by symphony-adnane — APPROVED** [review]

File: src/components/crop-content/CropContent.tsx (hunk 1 above)
Comment (AliciaSymphony) [reply src/components/crop-content/CropContent.tsx:14]: I think that both still make sense, initCollapsed is just at the initialisation, but collapsed you need to provide the logic to change it

**Comment by AliciaSymphony** [comment]
> I'd call the documentation button "toggle" instead of reverse XD

Thanks! Somehow I missed your comments, I pushed the changes in a new PR https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-components/pull/345

**Pull request merged**